test_parser.py

```diff
@@ -123,8 +120,7 @@ def test_parser(zone, data_type, target_datetime):
             elif data_type == "exchange":
                 validate_exchange(event, zone)
         except ValidationError as e:
-            print("Validation failed: {}".format(e))
-            sys.exit(3)
+            logger.warning('Validation failed @ {}: {}'.format(event['datetime'], e))
```

Comment:
Should we do a `raise` after the log here?

Reply:
I'd prefer not. The reason is that each datapoint is validated and might raise an error. Failing at the first datapoint would be a shame as we can't get an overview of which datapoint are failing validation.
Note that a parser returns multiple datapoints.